azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -146,6 +147,7 @@
   private final FlowTriggerScheduler scheduler;
   private final FlowTriggerService flowTriggerService;
   private Map<String, TriggerPlugin> triggerPlugins;
+  private final ExecutionLogsCleaner executionLogsCleaner;
 
   @Inject
   public AzkabanWebServer(final Props props,
```

Comment:
Does it matter that we are changing the Ctor signature?

Reply:
It doesn't matter. It will just be fine.